offchain-modules/lib/src/relay/eth_relay.rs

```diff
@@ -313,18 +226,24 @@ impl ETHRelayer {
                 continue;
             }
 
-            let from_lockscript = self.generate_from_lockscript(self.secret_key)?;
+            let from_privkey = self.secret_key;
+            let from_lockscript = self.generate_from_lockscript(from_privkey)?;
+
             let unsigned_tx = self.generator.generate_eth_light_client_tx(
                 &headers,
                 &cell,
                 &witnesses,
                 &un_confirmed_headers,
                 from_lockscript,
             )?;
-            let tx = sign(
+            // FIXME: waiting for sign server.
+            let secret_key_a = parse_privkey_path("0", &self.config, &Option::None)?;
```

Comment:
should pass the private keys in args, so we can test different situation with cli.